skfs/src/Context.sk

```diff
@@ -1626,4 +1626,31 @@ fun withRegionValue<T>(f: () ~> T): T {
   copied.getHead();
 }
 
+fun withRegionIterate<K, T>(
```

Comment:
Since `fn` takes and returns an accumulator, this more of a `fold` than an `iterate`.

Reply:
Ah, I was immediately comfortable here, follows the same naming pattern as e.g. https://clojuredocs.org/clojure.core/iterate

Reply:
I take this back. I misread the fun sig. It's a fold. :)